kong/runloop/plugin_servers/init.lua

```diff
@@ -352,15 +362,21 @@ end
 
 
 function plugin_servers.start()
-  if worker_id() ~= 0 then
-    return
+  if worker_id() == 0 then
+    local pluginserver_timer = proc_mgmt.pluginserver_timer
+
+    for _, server_def in ipairs(proc_mgmt.get_server_defs()) do
+      if server_def.start_command then
+        native_timer_at(0, pluginserver_timer, server_def)
```

Comment:
There is already a `timer.at` call in `proc_mgmt.pluginserver_timer`,
Is something wrong?